app/grandchallenge/notifications/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+from grandchallenge.notifications.models import Notification
+
+
+class NotificationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Notification
+        fields = "__all__"

```

Comment:
It's better to be explicit about what fields you're serializing. When using all `"__all__"`, if a developer adds a really private field on the model then it will be delivered to the user over the rest API! The same goes for Django forms, or django create/update views. Always set `fields = ("this", "that", etc ...)`.

Reply:
We should find if there is a way to automate a check like that...

Reply:
Ah yes, that makes sense. Now I'm wondering whether the user and action fields of the notification are necessary to include in the serializer. All we really need to update is the 'read' field, right?

Reply:
> We should find if there is a way to automate a check like that...

By that you mean checking that model serializers don't use `__all__` in their fields specification?

Reply:
Yes to both of those questions.